auto_backup/__openerp__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>). All Rights Reserved
+#    $Id$
+#    Copyright (C) 2015 Agile Business Group <http://www.agilebg.com>
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
```

Comment:
This is something I did not want to change for you, but still you say that it is licensed under AGPL, but this header points to GPL.

Please use AGPL headers (the [reduced OCA style](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3) ones if possible, please).

Also you need a merge to solve conflicts.